tests/utests/utests.h

```diff
@@ -648,6 +758,7 @@ struct utest_context {
  *                     CHECK_LYD_VALUE_ ## TYPE_VAL.
  */
 #define CHECK_LYD_VALUE(NODE, TYPE_VAL, ...) \
+    assert_ptr_equal((NODE).realtype->plugin, &(ly_builtin_type_plugins[LY_TYPE_ ## TYPE_VAL]));\
```

Comment:
May work for now, but once plugins are properly implemented, it most likely will not.